gallery/src/pages/Accessibility.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+import { NxP, NxCode, NxH3, NxList, NxTextLink } from '@sonatype/react-shared-components';
+
+const Accessibility = () =>
+  <GalleryTile title="Accessibility">
+    <NxP>
+      Accessibility (A11y) means ensuring that our applications are usable by everybody. We do this in a variety of
+      different ways. We make sure that our designs are clear and that they follow accessibility guidelines for
+      contrast and readability. Our components are written with semantic HTML, correct use of <NxCode>aria</NxCode>
+      tags, and testing in VoiceOver and ChromeVox accessibility is built into the Shared Components. However there is
+      still work that needs to be done to make sure that the application as a whole is accessible.
+    </NxP>
+
+    <NxH3 className="nx-tile__section-header">Making your pages accessible</NxH3>
+    <NxP>
+      Ensuring you use the components in an accessible manner is important, here are a few tips to help you make your
+      pages accessible.
+    </NxP>
+
+    <NxList className="nx-list--bulleted">
+      <NxList.Item>Tab order is important. Make sure that the tab order of the page is logical.</NxList.Item>
```

Comment:
We should mention and/or link to the `tabindex` documentation that talks about how you shouldn't use tabindex's greater than 0

Reply:
> We should mention and/or link to the `tabindex` documentation that talks about how you shouldn't use tabindex's greater than 0

https://github.com/sonatype/sonatype-react-shared-components/pull/455/commits/14deafa38e3c6c5fb8f70fc8b4860d82a57a14f7

Reply:
> 14deafa

Maybe put some emphasis on the part about _avoiding_ the higher numbers

Reply:
> > [14deafa](https://github.com/sonatype/sonatype-react-shared-components/commit/14deafa38e3c6c5fb8f70fc8b4860d82a57a14f7)
> 
> Maybe put some emphasis on the part about _avoiding_ the higher numbers

Well if a warning box was good enough for MDN it's good enough for us: https://github.com/sonatype/sonatype-react-shared-components/pull/455/commits/1046b1c52831da1aa0d54a038c1cfb8b043a5ed6